docs/OTAnalytics/gettingstarted/installation.md

```diff
@@ -2,7 +2,9 @@
 
 ## Install Python 3.9
 
-Install the 64-bit version Python 3.9.x via Windows installer from www.python.org/downloads/ as follows (Python 3.6.x to 3.8.x should also work, the 32-bit version is not supported):
+(If not done yet)
+
+Install the 64-bit version Python 3.9.x via Windows installer from www.python.org/downloads/ as follows (Python 3.8.x should also work, the 32-bit version is not supported):
```

Comment:
@hgoerner  pleas either add an "install-py38.bat" with the correct dependecies for GDAL and Fiona to the OTAnalytics repository or delete the phrase that implies Py38 support here in the Docs. Thanks!